src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.ts

```diff
@@ -54,7 +54,8 @@ export class UBSOrderDetailsComponent extends FormBaseComponent implements OnIni
   object: {};
   private destroy: Subject<boolean> = new Subject<boolean>();
   private destroyed$: ReplaySubject<any> = new ReplaySubject<any>(1);
-  public currentLanguage: string;
+  private langSub: Subscription;
+  public currentLanguage: string = localStorage.getItem('language');
```

Comment:
we should use localStorageSerrvice for this

Reply:
done